### DISCUSSION TITLE
Using wood to speed up cooking

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#3404] [Ideas]
Hello!

This proposal has been discussed for a while now in various chats but I will compile it here having talked to Aeon about it. It was discussed at great length around the time composter speed-up using eggs was introduced.

Essentially, use wood to speed up the cooking time of a building.

Boost per building:
Firepit - 15% quicker for 24h by adding 25 wood.
Kitchen - 20% quicker for 24h by adding 75 wood.
Bakery - 25% quicker for 24h by adding 150 wood.
Deli -  30% quicker for 24h by adding 200 wood.
Smoothie Shack - No wood for this? or yes? 

^ Simplest way to have it IMO. Also open to have it so that there is a minimum input/maximum boost to get if you want to use MORE wood than stated, although it would be more complicated and take longer to implement.

This would help with players wanting to use wood to speed up their cooking (level up faster) or help them to complete chores in time.

One thing to note is how this would work with Luna's Hat holders.
If a player with LH starts cooking a cake (base 24h), it drops to 12 hours. Then applying a 25% faster boost using wood, it's 25% off the 12 hours meaning it goes down to 9 hours

Please let me know if the overall idea is good and how those values look... 450 wood per day is the cost to have all those buildings boosted 24/7.

**Comment by rasheed19871** [comment]
I am totally agree with your proposal this is great dcol since now we have to cook 4 cakes (3 carrot 1 cabbage) as many people know that there is another category of players that exists in sunflower land which are not whales but they like the game is much as everyone else If doing this is hard for the team then I suggest we need another BACKERY come on Adam don't let this knock us from competition
